strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-maven-layout-provider/src/main/java/org/carlspring/strongbox/event/artifact/MavenArtifactDeletedEventListener.java

```diff
@@ -20,7 +21,7 @@
 
 
     @Inject
-    private MavenIndexGroupRepositoryComponent mavenIndexGroupRepositoryComponent;
+    private Optional<MavenIndexGroupRepositoryComponent> maybeMavenIndexGroupRepositoryComponent;
```

Comment:
Why are all these fields now prefixed with `maybe*`? When you're looking for the field/method to use, you won'd be expecting them to be prefixed this way.